package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "@observation.org/react-native-components",
-  "version": "1.42.0",
+  "version": "1.43.0",
```

Comment:
Weet niet of het echt nodig is, maar voor de zekerheid ook maar de version verhoogd.